chsdi/templates/htmlpopup/gletscherausdehnung.mako

```diff
@@ -0,0 +1,10 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c,lang)">
+<%
+  layer = 'ch.swisstopo.geologie-gletscherausdehnung.'
+%>
+<tr><td class="cell-left">${_(layer + 'name')}</td> <td>${c['attributes']['name'] or '-'}</td></tr>
+<tr><td class="cell-left">${_(layer + 'sgi-id')}</td> <td>${c['attributes']['sgi_id'] or '-'}</td></tr>
+<tr><td class="cell-left">${_(layer + 'area_km2')}</td> <td>${c['attributes']['area_km2'] or '-'}</td></tr>
```

Comment:
It would look nicer when rounded to a few digits. It's probably the customers decision though.